src/ui/components/UploadScreen/UploadScreen.tsx

```diff
@@ -87,19 +87,9 @@ function Actions({ onDiscard, status }: { onDiscard: () => void; status: Status
 
 function LimitWarning() {
   return (
-    <div className="absolute bottom-4 right-4 flex rounded-full bg-gray-500 p-2 text-white shadow-lg">
-      <PortalTooltip
-        tooltip={
-          <div
-            className="mb-4 rounded-md bg-toolbarBackground p-2 px-3 text-base shadow-lg"
-            style={{ width: "200px" }}
-          >
-            {`Replays work best when under 2 minutes`}
-          </div>
-        }
-      >
-        <MaterialIcon className="select-none">warning</MaterialIcon>
-      </PortalTooltip>
+    <div className="absolute place-content-center bottom-2 right-2 flex rounded-md bg-gray-500 p-2 text-white shadow-lg text-xs">
+      <MaterialIcon className="select-none">warning</MaterialIcon>{" "}
```

Comment:
Small nit, but I'd take out the `${" }` on line 91 and just add a `space-x-2` or something in the parent div on line 90.

Reply:
Totally agree, thanks!